CMakeLists.txt

```
@@ -100,6 +96,10 @@ else()
     message(FATAL_ERROR "Unknown or unsupported processor: " ${CMAKE_SYSTEM_PROCESSOR} ". Override by setting OQS_PERMIT_UNSUPPORTED_ARCHITECTURE=ON")
 endif()
 
+if (NOT (CMAKE_SYSTEM_NAME MATCHES "Linux|Darwin") AND (OQS_LIBJADE_BUILD STREQUAL ON) AND (ARCH_X86_64 STREQUAL "ON"))
```

I think this condition is not quite right: for instance, it seems like `-DOQS_LIBJADE_BUILD=ON` would make it through without a warning on Windows ARM builds.

---

```suggestion
if (NOT ((CMAKE_SYSTEM_NAME MATCHES "Linux|Darwin") AND (ARCH_X86_64 STREQUAL "ON")) AND (OQS_LIBJADE_BUILD STREQUAL "ON"))
```
I think this does the trick